parflow_data_management/__init__.py

```diff
@@ -1,7 +1,14 @@
+from .celery import app as celery_app
+
+# This will make sure the app is always imported when
+# Django starts so that shared_task will use this app.
+__all__ = ('celery_app',)
```

Comment:
Was there are reason you chose to use celery rather than dramatiq?

Reply:
Lack of documentation with dramatiq, especially for integration with Django. If you prefer dramatiq, I could certainly make it work. Celery just seemed like the safer option after attempting an implementation with dramatiq.

Reply:
No, I was just interested, thanks for letting me know.